src/strategies/terraform-module.ts

```diff
@@ -88,6 +89,26 @@ export class TerraformModule extends BaseStrategy {
         }),
       });
     });
+
+    // Update metadata.yaml and metadata.display.yaml to current candidate version.
+    const metadataFiles = await Promise.all([
+      this.github.findFilesByFilenameAndRef(
```

Comment:
Does this method return an empty array if no files are found for the search?

Reply:
Honestly, I can't tell looking at the source. I'm also a TS novice so I might be missing it completely. But I've wrapped the call in a try/catch so even if it errors out, we'll set `metadataFiles` to an empty array.